.github/workflows/core_build_no_pch.yml

```diff
@@ -0,0 +1,41 @@
+name: core-build-no-pch
+on:
+  push:
+    branches:
+      - 'master'
+  pull_request:
+
+jobs:
+  no-pch-build:
+    strategy:
+      fail-fast: false
+      matrix:
+        # the result of the matrix will be the combination of all attributes, so we get os*compiler*modules builds
+        os: [ubuntu-latest]
+        compiler: [clang]
+    runs-on: ${{ matrix.os }}
+    name: ${{ matrix.compiler }}
+    env:
+      COMPILER: ${{ matrix.compiler }}
+    steps:
+      - uses: actions/checkout@v2
+      - name: Cache
+        uses: actions/cache@v2
+        with:
+          path: /home/runner/.ccache
```

Comment:
```suggestion
          path: var/ccache
```

Reply:
@Yehonal built in support for ccache directly into the scripts so to properly save the cache you need to change the path to this.